hack/dockerfiles/test.Dockerfile

```diff
@@ -188,6 +188,7 @@ ENV HOME /home/user
 ENV USER user
 ENV XDG_RUNTIME_DIR=/run/user/1000
 ENV TMPDIR=/home/user/.local/tmp
+ENV BUILDKIT_HOST=unix:///run/user/1000/buildkit/buildkitd.sock
```

Comment:
why is this required now?

Reply:
`docker exec -i bk-rootless buildctl dial-stdio` needs valid `BUILDKIT_HOST` to be set in the bk-rootless container